<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>Research | Muhle Lab</title>
        <link rel="icon" type="image/x-icon" href="../icons/favicon.png">
        <link rel="stylesheet" href="../style.css">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
        <script src="../nav.js" defer></script>
    </head>

    <body>
        <div id="main">
            <div class="lab-banner research">
                <div class="header">
                    <div class="logo"><a href="../index.html" title="Home"><strong>Muhle</strong> Lab</a></div>
                    <i class="fa fa-bars mobile-nav-toggle" aria-controls="navbar" aria-expanded="false"><span class="sr-only">Menu</span></i>                    <div class="navbar" data-visible="false">
                        <a class="header-link"
                        href=".././index.html">Home</a>
                        <div class="dropdown">
                            <span class="header-link dropdown-button">Team</span>
                            <div class="dropdown-content" menu-visible="false">
                                <a class="header-link" href="./team.html">Members</a>
                                <a class="header-link" href="./alumni.html">Alumni</a>
                            </div>
                        </div>
                        <a class="header-link current-page"
                        href="./research.html">Research</a>
                        <a class="header-link" href="./news.html">News</a>
                        <a class="header-link"
                        href="./funding.html">Funding</a>
                        <a class="header-link"
                        href="./contact.html">Contact</a>
                    </div>
                </div>
                <div class="banner-content">
                    <h1>Research</h1>
                </div>
            </div>

            <div class="research-wrapper">
                <h2>Research Goals</h2>
                <div class="research-overview"> 
                    <p>Disruptive mutations in the gene <strong><em>CHD8</em></strong> (<b><em><u>C</u>hromodomain <u>H</u>elicase
                    <u>D</u>NA-binding protein <u>8</u></em></b>) are highly
                    associated with a diagnosis of autism spectrum disorder
                    (ASD) and other neurodevelopmental disorders. These changes
                    often result in the disruption of one copy of <i>CHD8</i>,
                    leading to less <i>CHD8</i> protein expression in developing
                    tissues such as the neocortex.</p>
                    <figure class="hchd8-dnm-diagram">
                        <img src="../images/hchd8-dnm-diagram.png" alt="figure of de novo
                        mutations in human CHD8 from patients diagnosed with ASD">
                        <figcaption>Schematic diagram of the human <i>CHD8</i> gene and position of likely damaging mutations reported in carriers diagnosed with ASD.</figcaption>
                    </figure>
                    <div class="chd8-histone-container">
                        <img class="chd8-histone" src="../images/chd8-histone.png" alt="diagram
                        illustrating rearrangement of nucleosomes via CHD8 binding
                        to methyl groups on histones"> 
                        <p><i>CHD8</i> binds to histone 3 lysine 4 (H3K4) methyl
                        groups and remodels (or moves) nucleosomes along the DNA. By
                        changing the accessibility of genomic regions, important
                        regulatory regions may be revealed, such as transcription
                        factor binding sites (TFBS) at promoter sites. In prior
                        studies with the Noonan lab at Yale University, we performed chromatin-immunopreciptation assays followed by high-throughput sequencing (ChIP-seq), to determine the localization of <i>CHD8</i> in neural stem cells, mouse cortex, and prenatal human brain. We found that <i>CHD8</i> binds most often to promoter regions of active genes. <i>CHD8</i> consistently bound gene targets across samples, including human neural stem cells, mouse cortex, and prenatal human brain.</p>
                    </div>
                    <figure class="chip-seq">
                        <img src="../images/chip-seq.jpg">
                        <figcaption>ChIP-seq of hNSCs and visualization using UCSC genome browser shows the overlap of <i>CHD8</i> binding (black) at the <i>DYRK1A</i> promoter (a known <i>CHD8</i> target and ASD risk gene) with active chromatin mark H3K27ac (green). Statistically significant regions (“peaks”) are indicated by the bars.</figcaption>
                    </figure>
                    <div class="asd-risk-network-container">
                        <img class="asd-risk-network" src="../images/asd-risk-network.jpg">
                        <p>Additionally, we found that <i>CHD8</i> regulates the expression of other genes associated with the likelihood of autism during neurodevelopment, and that these genes are co-expressed in networks that are also associated with likelihood of autism. Disruption of <i>CHD8</i> function may therefore alter the expression of the genes in these networks. Our research aims to characterize the molecular and circuit-level changes associated with loss of <i>CHD8</i> function. Specifically, we seek to understand how loss of <i>CHD8</i> affects genes, cell types, and regulatory networks critical for neurodevelopment.</p>
                    </div>
                </div>
                <hr class="hr-research top">
                <h2>Research Projects</h2>
                <div class="research-aim gif">
                    <figure class="chd8-ihc-gif">
                        <img
                        src="../images/chd8-ihc-male-wt.gif">
                        <figcaption>Fluorescent immunohistological staining of
                        <i>Chd8</i> (red) in the developing cortex of male WT
                        mice at various embryonic timepoints. Images prepared by
                        Martina Krenzer, MD.</figcaption>
                    </figure>
                    <div class="research-aim-text">
                        <h3>Which specific cell types show <i>Chd8</i> expression?</h3>
                        <p>Previous research conducted with the Noonan lab found regional expression of <i>Chd8</i> is dynamic with embryonic age in mice. The uniform expression across the neocortex at E12.5 gives way to increasingly regionalized expression in the mature neurons of the developing mouse cortex, including cells of the cortical plate, the subplate, and the mantle zone. We are employing single cell analysis in developmental timepoints spanning corticogenesis, using a heterozygous <i>Chd8</i><sup>+/-</sup> mouse line to determine the genome-wide expression and epigenetic profiles that define distinct brain cell populations in the context of <i>Chd8</i> loss. Additionally, we are examining a role for <i>Chd8</i> in non-cortical brain structures.</p>
                    </div>
                </div>
                <hr class="hr-research">
                <div class="research-aim reverse">
                    <figure class="chd8-tag">
                        <img
                        src="../images/chd8tag_mouse.png">
                        <figcaption>Experimental outline of the <i>in-vivo</i>
                        inducible biotinylation system we are using to map
                        <i>Chd8</i> regulatory networks at cellular resolution.</figcaption>
                    </figure>
                    <div class="research-aim-text">
                        <h3>Which genes are regulated by <i>Chd8</i> in specific cell types? </h3>
                        <p>We have generated a transgenic <i>Chd8</i>-Avi-tagged mouse line that allows for cell-type specific biotinylation by a Cre-activated biotin ligase, BirA. We are performing streptavidin-mediated ChIP-seq (bioChIP-seq) in tissues expressing activated BirA in cell-types of interest using Cre driver lines to target biotinylation. This will generate chromatin maps of distinct cell types to assess genes and biological pathways regulated by <i>Chd8</i>.</p>
                    </div>
                </div>
                <hr class="hr-research">
                <div class="research-aim">
                    <figure class="chd8-hnsc">
                        <img src="../images/chd8-hnsc.jpg">
                        <figcaption>By expressing an mCherry-tagged <i>CHD8</i> reporter in human neural stem cells, we will determine compounds that increase or decrease <i>CHD8</i> expression in-vitro.</figcaption>
                    </figure>
                    <div class="research-aim-text">
                        <h3><i>CHD8</i> function in human neural stem cells, a model of early neurodevelopment</h3>
                        <p>What are the negative and positive regulators of <i>CHD8</i> expression? We are studying the effects of bioactive compounds such as hormones on <i>CHD8</i> expression by using an mCherry-tagged <i>CHD8</i> reporter in hNSCs.</p>
                    </div>
                </div>
                <hr class="hr-research">
                <div class="research-aim reverse">
                    <figure class="straub-tail">
                        <img src="../images/straub-tail.jpeg">
                        <figcaption>A spontaneous seizure triggered during cage change in which a mouse is exhibiting tonic immobility, staring spell, and Straub tail sign.</figcaption>
                    </figure>
                    <div class="research-aim-text">
                        <h3>A long-term consequence of <i>Chd8</i> haploinsufficiency: Spontaneous seizures in <i>Chd8</i><sup>+/-</sup> mice</h3>
                        <p>Some people who carry a <i>CHD8</i> mutation have a
                        history of seizures or abnormal EEG, but this potential
                        co-morbidity is not well characterized in human cohorts
                        or model systems. We have found that seizure
                        susceptibility increases with age in
                        <i>Chd8</i><sup>+/-</sup> mice. What are the brain
                        regions and circuits responsible for these seizures? Identifying molecular markers preceding abnormal brain
                        activity in our <i>Chd8</i><sup>+/-</sup> mice may help
                        to understand the etiology and progression of the
                        seizure phenotype seen in patients with damaging
                        <i>CHD8</i> mutations.</p>
                    </div>
                </div>  
            </div>

            <div class="footer">
                <div class="wrapper">
                    <div class="logo"><strong>Muhle</strong> Lab</div>
                    <div class="footer-address">
                        <hr>
                        <p>
                            New York State Psychiatric Institute<br>
                            Columbia University Department of Psychiatry<br>
                        </p>
                    </div>
                    <div class="footer-address">
                        <hr>
                        <p>
                            Herbert Pardes Building, Room 4935<br>
                            1051 Riverside Dr<br>
                            New York, NY 10032<br>
                        </p>
                    </div>
                    <div class="footer-address">
                        <hr>
                        <p>
                            Copyright &copy; <script>document.write(new Date().getFullYear())</script> Muhle Lab
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </body>
</html>