<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>Research – Muhle Lab</title>
        <link rel="stylesheet" href="../style.css">
    </head>

    <body>
        <div class="lab-banner">
            <div class="header">
                <div class="logo"><a href="../index.html"><strong>Muhle</strong> Lab</a></div>
                <ul>
                    <li><a href="../index.html">Home</a></li>
                    <li><a href="./team.html">Team</a></li>
                    <li>Research</li>
                    <li>Funding</li>
                    <li><a href="./contact.html">Contact</a></li>
                </ul>
            </div>
            <div class="banner-content">
                <h1>Research</h1>
            </div>
        </div>

        <div class="wrapper">
            <div class="research-overview">
            <h2>Research Overview</h2>
            <p>Disruptive mutations in the gene <em>CHD8</em> (<em><u>C</u>hromodomain <u>H</u>elicase
            <u>D</u>NA-binding protein <u>8</u></em>) are highly associated with risk of
            autism spectrum disorder (ASD)—more than a dozen damaging mutations
            have been identified in <i>CHD8</i> from unrelated individuals
            diagnosed with ASD (<b>Fig. 1</b>).</p>
            
            <figure>
                <img class="hchd8-dnm-diagram" src="../images/hchd8-dnm-diagram.png" alt="figure of de novo
                mutations in human CHD8 from patients diagnosed with ASD">
                <figcaption><b>Figure 1.</b> Schematic diagram of the human
                <i>CHD8</i> gene and likely damaging mutations reported in medical
                literature of patients
                diagnosed with ASD.</figcaption>
            </figure>

            <p><i>CHD8</i> affects DNA accessibility of
            gene promoters by binding to H3K4 methyl groups and remodeling
            nucleosomes (<b>Fig. 2</b>). Additionally, <i>CHD8</i>
            regulates expression of ASD risk-associated networks during neurodevelopment, and disruption
            of <i>CHD8</i> function can alter expression of the genes in these
            networks. While <i>CHD8</i> function has been largely studied in
            the cortex, its role in other brain structures has been largely unexplored. Our research aims to
            characterize the molecular and circuit-level changes associated with
            loss of <i>CHD8</i> function. Specifically, we are interested to
            understand how loss of <i>CHD8</i> affects genes, cell types, and
            regulatory networks critical for neurodevelopment.</p>

            <figure class="figure-2">
                <img class="derubeis-nature-2014"
                src="../images/derubeis-nature-2014.png" alt="diagram
                illustrating CHD8 binding to H3K4 methyl groups on histones">
                <figcaption><b>Figure 2.</b> Diagram illustrating how
                <i>CHD8</i> (boxed in red) binds to H3K4 methyl groups when
                remodeling histones. De Rubeis et al., <i>Nature</i> 2014.</figcaption>
            </figure>
            </div>
            
            <div class="research-aims">
                <h3>Determine genomic regions and cell types altered by loss of <i>CHD8</i></h3>
                <p>We aim to identify the effects of <i>Chd8</i> reduction
                on cell types and lineages in the developing mouse brain. Using snRNAseq and snATACseq analysis in embryonic and
                postnatal developmental timepoints, we are determining the
                genome-wide expression and epigenetic profiles that define the distinct
                brain cell populations in WT and
                <i>Chd8</i><sup>+/-</sup> mice.</p>

                <h3>Generate chromatin maps of distinct cell types to
                assess genetic pathways regulated by <i>CHD8</i></h3>
                <p>We are using an <i>in vivo</i> inducible biotinylation system
                to map the <i>CHD8</i> regulatory networks at cellular
                resolution. We have generated a <i>Chd8</i>-tag mouse line in
                which a FLAG-Avi epitope is incorporated at the 3' end of the
                endogenous <i>Chd8</i> gene to encode a Chd8-tag protein.
                Activation of biotin ligase BirA using cell type specific Cre
                drivers of the developing brain will biotinylate Chd8-tag
                only in those cell types. Biotin-streptavidin pulldowns followed
                by bioChIPseq allows us to map Chd8 binding targets at cellular
                resolution. We are also performing conventional RNAseq and
                ChIPseq by isolating cells expressing GFP activated by the same
                Cre drivers to further define the transcriptional and epigenetic
                state of the cells used in bioChIPseq.</p>
            </div>
        </div>

        <div class="footer">
            <div class="wrapper">
                <div class="logo"><strong>Muhle</strong> Lab</div>
                <div class="footer-address">
                    <hr>
                    <p>
                        New York State Psychiatric Institute<br>
                        Columbia University Department of Psychiatry<br>
                    </p>
                </div>
                <div class="footer-address">
                    <hr>
                    <p>
                        Herbert Pardes Building, Room 4935<br>
                        1051 Riverside Dr<br>
                        New York, NY 10032<br>
                    </p>
                </div>
            </div>
        </div>
    </body>
</html>